### PR TITLE
Fix ballot parsing

### PR DESCRIPTION
**Pull request by av-lukas**

### CODE REVIEW

**Comment by av-emil** [comment]
Does this save the problem we talked about with the JS client failing to poll?

**Comment by av-lukas** [comment]
It makes the problem not happen anymore, but I don't have a scientific explanation for why that is. The type assertions were breaking the flow somehow.

File: yarn.lock [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
 
 
-"@aion-dk/js-client@1.1.3":
-  version "1.1.3"
-  resolved "https://registry.npmjs.org/@aion-dk/js-client/-/js-client-1.1.3.tgz"
-  integrity sha512-XGPZDLcHuFpu8lQllGph44G3aVUojBUBJdXDhjU1Ex4OHLFqEEmgsHtKOGoeQniaWwHpwgZSXnkLMch968/ENA==
+"@aion-dk/js-client@3.0.1":
+  version "3.0.1"
+  resolved "https://registry.yarnpkg.com/@aion-dk/js-client/-/js-client-3.0.1.tgz#00d7d092d8619fc5acc36730ce9d09646bc0c641"
+  integrity sha512-hga7lYCWo4Y3cYXaIHHcc2vo7fVptLZ/f427eV5C2TDgr9ZC9jyjVatOdI6zpKLzoPbQbJ7e3w8Oc7+LP/NHLw==
```
Comment (av-emil) [inline yarn.lock:8]: It seems you used yarn, while it was previously built with npm. Is this intentional?

**Review by av-emil — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (av-lukas) [reply yarn.lock:8]: This project has been using yarn for quite some time. Where do you see NPM being used here?

Comment (av-emil) [reply yarn.lock:8]: From the file diff: 
Screenshot 2023-04-28 at 09 07 48

Comment (av-lukas) [reply yarn.lock:8]: I see. I have no idea why that's the case.

**Review by av-emil — APPROVED** [review]

**Pull request merged**